CMSgov/design-system
# [WNMGDS-1914] Add a visual regression testing suite with Playwright

## Summary

https://jira.cms.gov/browse/WNMGDS-1914

- Adds a visual regression testing suite with Playwright

## Reviewing

[Here is a link to filter out the image files in the diff](https://github.com/CMSgov/design-system/pull/2142/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.lock&file-filters%5B%5D=.md&file-filters%5B%5D=.ts&file-filters%5B%5D=dotfile&show-viewed-files=true)

## How to test

1. `yarn install`
2. `yarn build:storybook`
3. `yarn test:browser:docker` - Needs to run in Docker on our machines for consistency. On Jenkins the whole job runs within the same docker container, so it runs with `yarn test:browser`
    - Feedback requested: Should I remove the `:docker` sub-command for consistency and have `yarn test:browser` always run in docker? Jenkins would then just have to run playwright directly with `yarn playwright test --config tests/browser/playwright.config.ts`

## Checklist
